src/masternodes/mn_rpc.cpp

```diff
@@ -4113,6 +4121,10 @@ static bool GetCustomTXInfo(const int nHeight, const CTransactionRef tx, CustomT
         case CustomTxType::AnyAccountsToAccounts:
             res = ApplyAnyAccountsToAccountsTx(mnview_dummy, ::ChainstateActive().CoinsTip(), *tx, nHeight, metadata, Params().GetConsensus(), true, &txResults);
             break;
+        case CustomTxType::AutoAuthPrep:
+            res.ok = true;
+            res.msg = "AutoAuth";
```

Comment:
`ToString(guess)` for consistency.

Reply:
Makes sense. Will change next time I'm in that area :)